docs/en/reference/basic-mapping.rst

```diff
@@ -423,36 +423,31 @@ the field that serves as the identifier with the ``#[Id]`` attribute.
             # fields here
 
 In most cases using the automatic generator strategy (``#[GeneratedValue]``) is
-what you want. It defaults to the identifier generation mechanism your current
-database vendor prefers: AUTO_INCREMENT with MySQL, sequences with PostgreSQL
-and Oracle and so on.
+what you want.
 
 .. _identifier-generation-strategies:
 
 Identifier Generation Strategies
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The previous example showed how to use the default identifier
-generation strategy without knowing the underlying database with
-the AUTO-detection strategy. It is also possible to specify the
-identifier generation strategy more explicitly, which allows you to
-make use of some additional features.
-
 Here is the list of possible generation strategies:
 
 -  ``AUTO`` (default): Tells Doctrine to pick the strategy that is
-   preferred by the used database platform. The preferred strategies
-   are IDENTITY for MySQL, SQLite, MsSQL and SQL Anywhere and SEQUENCE
-   for Oracle and PostgreSQL. This strategy provides full portability.
+   preferred by the used database platform:
```

Comment:
Looks like this is no longer true since https://github.com/doctrine/orm/pull/8870 : `AbstractPlatform::prefers*` methods have been removed. It's now more based on what the platforms supports: https://github.com/doctrine/orm/blob/da9b9de5906802cab1c71f59ffd9fba53ed4011d/lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php#L727-L745

Reply:
Related: https://github.com/doctrine/orm/pull/8931